Common/ConvertUproot.py

```diff
@@ -0,0 +1,47 @@
+import uproot
+import awkward as ak
+import os
+import numpy as np
+def saveFile(outFile, out_tree, treeName):
+  if os.path.exists(outFile):
+    os.remove(outFile)
+    with uproot.recreate(outFile, compression=uproot.LZMA(9)) as out_file:
+      out_file[treeName] = out_tree
+      out_file.close()
+    return out_file,outFile
+
+def toUproot(workingDir, fileName):
+  inFile = os.path.join(workingDir, fileName)
+  treeName = fileName.strip(".root")
+  input_file = uproot.open(f"{inFile}")
+  input_tree = input_file[f"{treeName}"]
+  keys = input_tree.keys()
+  outFile=f"{os.path.join(workingDir, treeName)}.root"
+  out_tree = {}
+  if not keys:
+    out_tree = {"noneCol":np.array([-1,-1,-1,-1])}
```

Comment:
Why? If the tree is empty just create an empty output file

Reply:
ok